app/models/setting.rb

```diff
@@ -175,6 +175,14 @@ def self.create!(opts)
     end
   end
 
+  def self.regexp_expand_wildcard_string(string)
+    "\\A#{Regexp.escape(string).gsub('\*', '.*')}\\Z"
+  end
+
+  def self.convert_array_to_regexp(array)
+    Regexp.new(array.map {|string| regexp_expand_wildcard_string(string) }.join('|'))
```

Comment:
This exception is correct, it was raised because I pushed a version without setting being renamed completely. If we make it pass even if the setting returns nil, it would just mask more serious issue - that the setting we rely on does not exist.